modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/modal/Modal.js

```diff
@@ -131,6 +135,12 @@ const Modal = ({
 		return <div className="liferay-modal-body" ref={bodyRef}></div>;
 	};
 
+	const ariaLabelledby = useMemo(() => {
+		modalLabelIdCounter++;
+
+		return `open-modal-label-${modalLabelIdCounter}`;
+	}, []);
+
```

Comment:
@diegonvs Do we need this, and the related `ariaLabelledby` logic? It seems like we are duplicating that will be added in Clay iteration, with https://github.com/liferay/clay/pull/4737. Clay update: https://github.com/liferay-frontend/liferay-portal/pull/2029

Reply:
Removed and squashed